packages/web3-providers-eip1193/src/errors.ts

```diff
@@ -0,0 +1,29 @@
+import { Web3Error, Web3PackageErrorConfig } from 'web3-core-logger/src/types';
+import packageVersion from './_version';
+
+export enum Web3ProvidersEip1193ErrorNames {
+    invalidClient = 'invalidClient',
+    noClient = 'noClient',
+}
+
+interface Web3ProvidersEip1193ErrorsConfig extends Web3PackageErrorConfig {
+    errors: Record<Web3ProvidersEip1193ErrorNames, Web3Error>;
+}
+
+export const Web3ProvidersEip1193ErrorsConfig: Web3ProvidersEip1193ErrorsConfig =
+    {
+        packageName: 'web3-providers-eip1193',
+        packageVersion,
+        errors: {
+            invalidClient: {
+                code: 1,
```

Comment:
As the error handling is currently, we don't really make use of `code`. I included it because I thought users may want to make use of it

Reply:
Removed `code`